### PR TITLE
added SafeMath.sol to Validator Registration contract

### LINKED ISSUE #1141: Add SafeMath.sol to Validator Registration Contract
Following #1110,  I add Safemath library to prevent uint underflow/overflow here:
uint timestampDayBoundry = block.timestamp - block.timestamp % SECONDS_PER_DAY + SECONDS_PER_DAY;

### PR DESCRIPTION
**Pull request by cyberbono3**
I add Safemath library to prevent uint underflow/overflow here:
uint timestampDayBoundry = block.timestamp - block.timestamp % SECONDS_PER_DAY + SECONDS_PER_DAY;

Resolves #1141

### CODE REVIEW

File: contracts/validator-registration-contract/SafeMath.sol [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    return c;
+  }
+
+  /**
+  * @dev Integer division of two numbers truncating the quotient, reverts on division by zero.
+  */
+  function div(uint256 a, uint256 b) internal pure returns (uint256) {
+    require(b > 0); // Solidity only automatically asserts when dividing by 0
+    uint256 c = a / b;
+    // assert(a == b * c + a % b); // There is no case in which this doesn't hold
```
Comment (prestonvanloon) [inline contracts/validator-registration-contract/SafeMath.sol]: Remove?

Why is this here if it can never happen?

**Comment by terenc3t** [comment]
Lint for solidity failed. Can we please fix it? @cyberbono3

**Comment by cyberbono3** [comment]
Sure, working on it

**Comment by cyberbono3** [comment]
Could you review, please?

File: contracts/validator-registration-contract/.soliumignore [hunk 2]
```diff
@@ -1 +0,0 @@
-node_modules
```
Comment (terenc3t) [inline contracts/validator-registration-contract/.soliumignore]: Why did you delete this? Can you revert?

Comment (prestonvanloon) [reply contracts/validator-registration-contract/.soliumignore]: +1, I don't think we have node_modules directory with bazel, but I still wonder what the motivation for this was.

Is it because we don't have node_modules with bazel?

**Comment by cyberbono3** [comment]
although PR is approved, it has not been merged with prysm/master, has it?

**Comment by terenc3t** [comment]
@cyberbono3 not yet, there's still an open item:

can you please revert the deletion of `contracts/validator-registration-contract/.soliumignore`?

**Comment by terenc3t** [comment]
@cyberbono3 something went wrong with the revert, the diff is now (+19,951 −3,385)
